documentation/book/assembly-kafka-bridge-overview.adoc

```diff
@@ -28,6 +35,20 @@ You can:
 
 * Seek on a partition, so that a consumer starts receiving messages from the first or last offset position, or a given offset position.
 
-Similar to a Kafka Connect cluster, you can deploy the Kafka Bridge into your Kubernetes cluster using the Cluster Operator. For deployment instructions, see xref:kafka-bridge-{context}[].
+Similar to a Kafka Connect cluster, you can deploy the Kafka Bridge into your Kubernetes cluster by using the Cluster Operator. For deployment instructions, see xref:deploying-kafka-bridge-str[].
```

Comment:
Is it needed to mention Kafka Connect here? We could mention Kafka Mirror Maker as well. I would mention neither.

Reply:
I suppose it might be confusing. I've removed the clause regarding Kafka Connect.